README.md

```diff
@@ -14,29 +39,33 @@ cd ubuntu-desktop-installer
 git submodule update --init --recursive
 ```
 
-Install and configure Flutter:
-```sh
-snap install flutter --classic
-flutter upgrade
-flutter config --enable-linux-desktop
-```
-
-Install dependencies:
+Install the Python (subitquity), Dart and Flutter dependencies:
 ```sh
 cd packages/subiquity_client
 dart pub get
 cd subiquity
 make install_deps
```

Comment:
The commands listed in the earlier comment should be enough, really. The one that runs `installdeps.sh` as discussed at https://github.com/canonical/ubuntu-desktop-installer/issues/458#issuecomment-966983273